azurerm/data_source_private_link_service.go

```diff
@@ -0,0 +1,474 @@
+						},
+						"enable_ip_forwarding": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+						"hosted_workloads": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"ip_configurations": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"name": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"location": azure.SchemaLocationForDataSource(),
+						"mac_address": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"network_security_group": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"location": azure.SchemaLocationForDataSource(),
+									"name": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"tags": tagsForDataSourceSchema(),
+									"type": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"primary": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+						"private_endpoint": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"location": azure.SchemaLocationForDataSource(),
+									"name": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"tags": tagsForDataSourceSchema(),
+									"type": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"resource_guid": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"tags": tagsForDataSourceSchema(),
+						"tap_configurations": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"name": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"type": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"virtual_machine": {
```

Comment:
could we flatten this to be just `virtual_machine_id`?

Reply:
Removed unnecessary field.